### PR TITLE
Bricks CLI Auth

### PR DESCRIPTION
**Pull request by fjakobs**
https://user-images.githubusercontent.com/40952/230129085-0e2e1d3d-2a92-46e7-b935-e10572e14ab9.mov

### CODE REVIEW

File: packages/databricks-sdk-js/src/config/BricksCliCredentials.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        }
+
+        const ts = this.getTokenSource(config);
+        try {
+            await ts();
+        } catch (e) {
+            if (e instanceof ConfigError) {
+                if (e.message.indexOf("Can't find 'bricks' command") >= 0) {
+                    config.logger.debug(
+                        "Most likely Bricks CLI is not installed"
```
Comment (nfx) [inline packages/databricks-sdk-js/src/config/BricksCliCredentials.ts:34]: please add a corner case when oauth is not configured or not supported

https://github.com/databricks/databricks-sdk-go/blob/main/config/auth_u2m.go#L32-L35

**Review by nfx — APPROVED** [review]
lgtm

File: packages/databricks-vscode/src/cli/CliWrapper.ts [hunk 2]
```diff
@@ -41,9 +41,13 @@ function getValidHost(host: string) {
...
+        return this.extensionContext.asAbsolutePath("./bin/bricks");
+    }
+
     getTestBricksCommand(): Command {
         return {
-            command: this.extensionContext.asAbsolutePath("./bin/bricks"),
+            command: this.bricksPath,
             args: [],
         };
     }
```
Comment (kartikgupta-db) [inline packages/databricks-vscode/src/cli/CliWrapper.ts:53]: We can remove this function

**Review by kartikgupta-db — APPROVED** [review]
LGTM

File: packages/databricks-vscode/src/cli/CliWrapper.ts (hunk 2 above)
Comment (fjakobs) [reply packages/databricks-vscode/src/cli/CliWrapper.ts]: aaaand it's gone

File: packages/databricks-sdk-js/src/config/BricksCliCredentials.ts (hunk 1 above)
Comment (fjakobs) [reply packages/databricks-sdk-js/src/config/BricksCliCredentials.ts:34]: Thanks, added.

**Pull request merged**